forc/src/cli/commands/test.rs

```diff
@@ -142,6 +147,21 @@ fn print_tested_pkg(pkg: &TestedPackage, test_print_opts: &TestPrintOpts) -> For
         // If logs are enabled, print them.
         if test_print_opts.print_logs {
             let logs = &test.logs;
+            if test_print_opts.decode_logs {
+                for log in logs {
+                    if let Receipt::LogData {
+                        rb,
+                        data: Some(data),
+                        ..
+                    } = log
+                    {
+                        let decoded_log_data = decode_log_data(*rb, data, &pkg.built.program_abi)?;
+                        let var_value = decoded_log_data.value;
+                        info!("Decoded log value: {}, log rb: {}", var_value, rb);
+                    }
+                }
+                info!("Raw logs:");
```

Comment:
Maybe we should have a separate function for printing this, like `raw_log(a)`?

For me, when I use `log(x)`, I'd expect it to only print the value of x.

Reply:
That is correct, the reason i did this way is because decoding logs are very primitive without var names etc, so that is also why this is under a flag `--decode` as well, I am fine with removing raw logs with decoded logs too. Maybe we should have `--raw`, if `--decode` and `--raw` is both provided we can show the current output, with only `--decode` we will only see `Decoded log value ...` lines.

Reply:
Created #6015